pyproject.toml

```diff
@@ -11,4 +11,8 @@ requires = [
 ]
 build-backend = "setuptools.build_meta"
 
+[tool.black]
```

Comment:
I still think this is a bad idea. Unless, the bits that hugely break readability are fixed first which would make sense to be separate from pyupgrade.